dashboard/app/controllers/pd/workshop_daily_survey_controller.rb

```diff
@@ -514,15 +530,37 @@ def render_csf_survey(survey_name, workshop)
       render :new_general
     end
 
-    def render_survey_foorm(survey_name:, workshop:, session:, day:)
+    def ayw_helper(survey_map, should_have_attended:, day:)
```

Comment:
This is pretty logically tricky and I don't se any unit tests, might be good to add as a follow up?